deploy/personal/README.md

```diff
@@ -0,0 +1,42 @@
+# Deployment
+
+Instructions on how to deploy app onto AWS Fargate on your **personal** AWS account
+
+## Pre-requisite
+
+-   Make sure you have installed [Terraform](https://learn.hashicorp.com/tutorials/terraform/install-cli), [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2-mac.html#cliv2-mac-prereq), and configured a `default` AWS CLI profile (see doc [here](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html#cli-configure-quickstart-profiles))
+
+```bash
+terraform -help # prints Terraform options
+which aws # prints /usr/local/bin/aws
+aws --version # prints aws-cli/2.0.36 Python/3.7.4 Darwin/18.7.0 botocore/2.0.0
+aws configure # configure your AWS CLI profile
+```
+
+## Configuration
+
+-   Create a Github project, and generate a personal access token (see doc [here](https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token))
+
+-   Create a secret on [AWS Secrets Manager](https://aws.amazon.com/secrets-manager/) named `DockerHubAccessToken` with key `DOCKER_HUB_ACCESS_TOKEN`, and your [Docker access token](https://docs.docker.com/docker-hub/access-tokens/) as value
+
+*   Populate `terraform.tfvars`:
+
+```bash
+default_region      = "us-east-1"
+docker_username     = "matlau"
+github_username     = "MatthewCYLau"
```

Comment:
Not very experienced in terraform - is it possible to configure these to be environment variables?

Reply:
nevermind... i see that's probably what `tfvars` is doing!

Reply:
Correct!